app/src/main/res/layout/onlinefeedview_header.xml

```diff
@@ -1,35 +1,108 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        xmlns:tools="http://schemas.android.com/tools"
-        android:orientation="vertical"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:paddingLeft="4dp"
+    android:paddingRight="4dp"
+    android:layout_marginTop="8dp">
+
+    <TextView
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_width="match_parent">
+        android:layout_marginStart="16dp"
+        android:layout_marginLeft="16dp"
+        android:layout_marginEnd="16dp"
+        android:layout_marginRight="16dp"
+        android:layout_marginBottom="8dp"
+        android:text="@string/about_podcast"
+        android:textSize="@dimen/text_size_semi_medium"
+        android:fontFamily="sans-serif-medium"
+        android:textFontWeight="600"
```

Comment:
Is there a style for this? Maybe even a default Material one from MaterialComponents?

Reply:
Should be a style instead of writing all the font attributes